executor/adapter.go

```diff
@@ -1480,15 +1480,40 @@ func resetCTEStorageMap(se sessionctx.Context) error {
 // LogSlowQuery is used to print the slow query in the log files.
 func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 	sessVars := a.Ctx.GetSessionVars()
+	// Disable recording the internal queries.
```

Comment:
@tonyxuqqi @zhangjinpeng1987 
Do you think it's necessary to disable recording the internal queries to make the diagnose more efficent?

Reply:
LGTM

Reply:
I'm afraid not as we may also need to diagnose internal queries...

Reply:
@MyonKeminta 
In this debugging scenario internal queries does not impact too much, for the common product form it's needed to be considered I think.

Reply:
Only limit the `inefficient` log to user queries, PTAL.